pkg/cli/admin/inspect/resource.go

```diff
@@ -166,6 +166,10 @@ func obtainRelatedObjects(obj *unstructured.Unstructured) ([]*configv1.ObjectRef
 	// obtain related namespace info for the current resource
 	klog.V(1).Infof("Gathering related object reference information for %q...\n", unstructuredToString(obj))
 
+	if obj.GetKind() == "ValidatingWebhookConfiguration" || obj.GetKind() == "MutatingWebhookConfiguration" {
```

Comment:
Nit: let's be explicit and move this after this block:https://github.com/openshift/oc/blob/0703a5baabaf66ca39604cfb655fd854d991b294/pkg/cli/admin/inspect/resource.go#L124-L127

This way we'll have clean `obtainReleatedObjects` and `obtainWebhookService` as two steps in `gatherRelatedObjects`. In the future, if someone decides to add `.status.relatedObject` to webhooks we'll still have this mechanism working.

Reply:
`obtainWebhookService` is a rename of your `obtainWebhookRelatedObjects`